meinberlin/apps/ideas/exports.py

```diff
@@ -31,3 +33,33 @@ def get_queryset(self):
             .annotate_comment_count()\
             .annotate_positive_rating_count()\
             .annotate_negative_rating_count()
+
+
+@register_export(_('Comments for ideas'))
+class IdeaCommentExportView(a4_export_mixins.ExportModelFieldsMixin,
+                            export_mixins.UserGeneratedContentExportMixin,
+                            a4_export_mixins.ItemExportWithLinkMixin,
+                            a4_export_mixins.ItemExportWithRatesMixin,
+                            export_mixins.ItemExportWithRepliesToMixin,
+                            a4_export_views.BaseItemExportView):
+
+    model = Comment
+
+    fields = ['id', 'comment', 'created']
+
+    def get_queryset(self):
+        comments = (Comment.objects.filter(idea__module=self.module) |
+                    Comment.objects.filter(
+                    parent_comment__idea__module=self.module))
```

Comment:
Just out of coriosity, that could also be written diffrently:

`Comment.objects.filter(Q(idea__module=module) |  Q(parent_comment__idea__module=module`

Confirming this:
```
>>> q2 = str((Comment.objects.filter(idea__module=module) |  Comment.objects.filter(parent_comment__idea__module=module)).query)
>>> q1 = str(Comment.objects.filter(Q(idea__module=module) |  Q(parent_comment__idea__module=module)).query)
>>> q1 == q2
True
```

I think that with set operations the django orm sometimes uses subqueries, wich are inneficient, but in that case he optimises that away, so its the same